packages/story-editor/src/app/highlights/states.js

```diff
@@ -28,17 +28,17 @@ import {
  * Highlight state object
  *
  * @typedef {Highlight} Highlight The current state of editor highlights
- * @property {string} tab The ID of the Inspector tab to highlight
+ * @property {string} tab The ID of the Sidebar tab to highlight
  * @property {boolean} focus Whether there is a focusable element, use for accessibility
  */
 
 const keys = {
   STORY_TITLE: 'STORY_TITLE',
 
-  // Inspector tabs
+  // Style tabs
```

Comment:
🤔  The three tabs at the top aren't the "style" section. I think this needs a better name so that it can be differentiated from the other content in the sidebar.

Having this be the "style tabs" would conflict with the "style panel" and the "style pane". Additionally, `Style` being one of the "style tabs" is confusing.

How about something like `Action tabs` or `Sidebar tabs`?

Reply:
It seems like the name that's been used throughout the PR is `sidebar tabs`, so that could work here too.

Reply:
Using `Sidebar tabs` 👍🏻